app/src/main/java/com/example/tmdb/ui/features/login/LoginScreen.kt

```diff
@@ -55,6 +56,7 @@ fun LoginScreen(
     }
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
```

Comment:
Use `@ExperimentalMaterial3Api` to make it consistent with other experimental annotations.

Reply:
Fixed in new commit